src/Tribe/REST/V1/Documentation/QR_Definition_Provider.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
missing namespace.

Reply:
This is pretty much old code, we don't use namespaces on these portions.